src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
```

Comment:
```suggestion
    public String getRandomColor() {
```
![image](https://github.com/mate-academy/jv-lottery/assets/12864128/ca7da8d1-e540-4438-a5b0-be68d3c19c8b)